collect-intent.rst

```diff
@@ -0,0 +1,145 @@
+Launching ODK Collect from External Apps
+==========================================
+
+:doc:`collect-guide` allows us to open several of its activities from another app. You can open a specific form or lists of empty forms, saved forms, finalized forms or sent forms. You can also build your own app that interacts with ODK Collect through intents. On the other side, you can also open other apps from ODK Collect.
+
+This doc describes how to launch ODK Collect and open its activities directly from an external app.
+
+.. note::
```

Comment:
This is too long for a note, and is information that is essential to understanding how to follow the tutorial. So it probably should just be a section like _About Intents_ or _Understanding Intents_ or something similar.